docs/user-guide/cli-using-using-environment-variables.md

```diff
@@ -1,19 +1,19 @@
 # Using environment variables
 
-**Note:** For information on how to modify Zowe CLI default environment variables, see [Configuring Zowe CLI environment variables](cli-configuringcli-ev.md).
+:::note
 
-You can define environment variables to execute commands more efficiently. Store a value such as your password in an environment variable, then issue commands without specifying your password every time. 
+For information on how to modify Zowe CLI default environment variables, see [Configuring Zowe CLI environment variables](cli-configuringcli-ev.md).
 
-The term "environment" can refer to your operating system, container environment, or automation server such as Jenkins.
+:::
 
-You may want to assign a variable in the following scenarios:
+You can define environment variables to execute commands more efficiently. Store a value such as your password in an environment variable, then issue commands without specifying your password every time. 
 
-* **Store a value that is commonly used.**
+The term *environment* can refer to your operating system, container environment, or automation server such as Jenkins.
 
-    For example, you might want to specify your mainframe username as an environment variable. Now you can issue commands and omit the `--user` option, and Zowe CLI automatically uses the value that you defined in the environment variable.
-* **Override a value in existing profiles.**
+Consider assigning a variable in the scenarios outlined in the following table.
 
-    For example, you might want to override a value that you previously defined in multiple profiles to avoid recreating each profile. Specify the new value as a variable to override the value in profiles.
-* **Secure credentials in an automation server or container.**
-    
-    You can set environment variables for use in scripts that run in your CI/CD pipeline. For example, can define environment variables in Jenkins so that your password is not seen in plaintext in logs. You can also define sensitive information in the Jenkins secure credential store.
\ No newline at end of file
+| Use case | Example | Benefit |
+| - | - | - |
+| Store a commonly used value. | Specify your mainframe username as an environment variable. | Issue commands without the `--user` option, and Zowe CLI automatically uses the value defined in the environment variable. |
+| Override a value in existing profiles. | Override a value previously defined in multiple profiles. Specify the new value as a variable to override the value in profiles. | Avoid recreating each profile. |
```

Comment:
@adam-wolfe -- I agree -- can you tell me the env variables? I'll add those to the chart.

Reply:
@adam-wolfe Would the example env var for the first use case be `ZOWE_OPT_USER`? Not sure about the second use case.

Reply:
Actually, it might be better to add a link on this page to https://docs.zowe.org/stable/user-guide/cli-using-formatting-environment-variables/ since that page describes how to use environment variables in place of command line options and provides examples.

Reply:
Disregard this comment thread. I did not realize that https://docs.zowe.org/stable/user-guide/cli-using-formatting-environment-variables/ appears directly under https://docs.zowe.org/stable/user-guide/cli-using-using-environment-variables in the sidebar.